extras/completions/floaty.bash

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+_vmfloaty()
+{
+  local cur prev subcommands template_subcommands hostname_subcommands
+  COMPREPLY=()
+  cur="${COMP_WORDS[COMP_CWORD]}"
+  prev="${COMP_WORDS[COMP_CWORD-1]}"
+
+  subcommands="delete get help list modify query revert snapshot ssh status summary token"
+  template_subcommands="get ssh"
+  hostname_subcommands="delete modify query revert snapshot"
+
+  if [[ $template_subcommands =~ (^| )$prev($| ) ]] ; then
+    if [ -z "$_vmfloaty_avail_templates" ] ; then
```

Comment:
Consider using the extended test (`[[`) instead of classic test (`[`); should be essentially a drop-in here.